### PR TITLE
Reduce time to execute tests by 10 minutes

### PR DESCRIPTION
**Pull request by cbickel**
Do we really need this test?
If yes, what about paralleling it?

### CODE REVIEW

**Comment by rabbah** [comment]
yes we need it because we had a bug where actions were not running for the full 5 minutes.

**Comment by rabbah** [comment]
note it doesn't run on travis, only internal jenkins.

**Comment by markusthoemmes** [comment]
Why don't we do this for swift?

**Comment by rabbah** [comment]
we could - iirc i explained this away as python proxy is the same one used for swift, so it was low risk because the bug/issues was from the node proxy.

**Comment by markusthoemmes** [comment]
Expected that answer.

Change is still LGTM. Parallelising this is goodness.

File: tests/src/whisk/core/limits/MaxActionDurationTests.scala [hunk 1]
```diff
@@ -43,15 +43,17 @@ class MaxActionDurationTests
     implicit val wskprops = WskProps()
     val wsk = new Wsk
 
-    Map("node" -> "helloDeadline.js", "python" -> "timedout.py", "java" -> "timedout.jar").foreach {
-        case (k, name) =>
-            s"$k action" should "run up to the max allowed duration" in withAssetCleaner(wskprops) {
-                (wp, assetHelper) =>
+    s"node-, python, and java-action" should "run up to the max allowed duration" in withAssetCleaner(wskprops) {
```
Comment (rabbah) [inline tests/src/whisk/core/limits/MaxActionDurationTests.scala]: nit: drop `s` no string interpolation here `s"node-, python, and java-action" `

**Comment by rabbah** [comment]
already pg'ed?

**Pull request merged**